src/components/UserSettings/GlobalSettings.js

```diff
@@ -5,52 +5,54 @@ import { connect } from 'react-redux'
 import { push } from 'connected-react-router'
 import { saveGlobalOptions } from '_/actions'
 import { FormControl, Switch } from 'patternfly-react'
-import { msg } from '_/intl'
-import localeWithFullName from '_/intl/localeWithFullName'
+import { withMsg, localeWithFullName } from '_/intl'
 import style from './style.css'
 
 import { Settings, SettingsBase } from '../Settings'
 import SelectBox from '../SelectBox'
 import moment from 'moment'
 
 class GlobalSettings extends Component {
-  dontDisturbList = [
-    {
-      id: 10,
-      value: moment.duration(10, 'minutes').humanize(),
-    },
-    {
-      id: 60,
-      value: moment.duration(1, 'hours').humanize(),
-    },
-    {
-      id: 60 * 24,
-      value: moment.duration(1, 'days').humanize(),
-    },
-    {
-      id: Number.MAX_SAFE_INTEGER,
-      value: msg.untilNextPageReload(),
-    },
-  ]
-
-  refreshIntervalList = [
-    {
-      id: 30,
-      value: msg.every30Seconds(),
-    },
-    {
-      id: 60,
-      value: msg.everyMinute(),
-    },
-    {
-      id: 120,
-      value: msg.every2Minute(),
-    },
-    {
-      id: 300,
-      value: msg.every5Minute(),
-    },
-  ]
+  dontDisturbList (msg) {
+    return [
+      {
+        id: 10,
+        value: moment.duration(10, 'minutes').humanize(),
+      },
+      {
+        id: 60,
+        value: moment.duration(1, 'hours').humanize(),
+      },
+      {
+        id: 60 * 24,
+        value: moment.duration(1, 'days').humanize(),
+      },
```

Comment:
@rszwajko Not related to this fix but I just noticed that those are not translated:
minutes, hours, days.
Please move them to messages.

Reply:
Yeah `translated-time-durations` based duration strings only handle translations in the `moment.duration.format()` call.  The `humanize` call is in moment itself, and the dummy locale data is not generated for moment based localization.